### PR TITLE
chore: support-bundle-kit version bump

### PR DESCRIPTION
**Pull request by c3y1huang**
Included fix for multi-platform

https://github.com/longhorn/longhorn/issues/2759
https://github.com/rancher/support-bundle-kit/pull/59

### CODE REVIEW

**Comment by c3y1huang** [comment]
@yangchiu `rancher/support-bundle-kit:v0.0.15` published.
Please help mirror to longhornio, thanks!

File: deploy/longhorn-images.txt [hunk 1]
```diff
@@ -10,4 +10,4 @@ longhornio/longhorn-instance-manager:v2_20221123
 longhornio/longhorn-manager:master-head
 longhornio/longhorn-share-manager:v1_20220914
 longhornio/longhorn-ui:master-head
-longhornio/support-bundle-kit:v0.0.14
\ No newline at end of file
+longhornio/support-bundle-kit:v0.0.15
```
[the remark this answers is not in the document]
Comment (yangchiu) [reply deploy/longhorn-images.txt:13]: @c3y1huang v0.015 mirrored! https://hub.docker.com/r/longhornio/support-bundle-kit/tags

**Review by innobead — APPROVED** [review]

**Comment by c3y1huang** [comment]
Manually testing on arm64 cluster with `longhornio/support-bundle-kit:v0.0.15` and `rancher/support-bundle-kit:v0.0.15` both crashlooped.
```
> kubectl -n longhorn-system get deployment/longhorn-support-bundle-manager-support-bundle-2022-11-30t11-40-38z -o yaml | grep image:
        image: rancher/support-bundle-kit:v0.0.15

> kubectl -n longhorn-system logs longhorn-support-bundle-manager-support-bundle-2022-11-30tkkbcj
+ exec tini -- /usr/bin/support-bundle-kit manager
/usr/bin/entrypoint.sh: line 4: /usr/bin/tini: cannot execute binary file: Exec format error
/usr/bin/entrypoint.sh: line 4: /usr/bin/tini: Success
```

Testing the same arm64 cluster with `rancher/support-bundle-kit:master-head` is working.

**Comment by yangchiu** [comment]
> Manually testing on arm64 cluster with `longhornio/support-bundle-kit:v0.0.15` and `rancher/support-bundle-kit:v0.0.15` both crashlooped.
> 
> ```
> > kubectl -n longhorn-system get deployment/longhorn-support-bundle-manager-support-bundle-2022-11-30t11-40-38z -o yaml | grep image:
>         image: rancher/support-bundle-kit:v0.0.15
> 
> > kubectl -n longhorn-system logs longhorn-support-bundle-manager-support-bundle-2022-11-30tkkbcj
> + exec tini -- /usr/bin/support-bundle-kit manager
> /usr/bin/entrypoint.sh: line 4: /usr/bin/tini: cannot execute binary file: Exec format error
> /usr/bin/entrypoint.sh: line 4: /usr/bin/tini: Success
> ```
> 
> Testing the same arm64 cluster with `rancher/support-bundle-kit:master-head` is working.

probably need this https://github.com/rancher/support-bundle-kit/pull/60

**Comment by yangchiu** [comment]
Temporarily mirrored `rancher/support-bundle-kit:master-head` to `longhornio/support-bundle-kit:v0.0.14` to let tests run.

File: deploy/longhorn-images.txt [hunk 2]
```diff
@@ -10,4 +10,4 @@ longhornio/longhorn-instance-manager:v2_20221130
 longhornio/longhorn-manager:master-head
 longhornio/longhorn-share-manager:v1_20220914
 longhornio/longhorn-ui:master-head
-longhornio/support-bundle-kit:v0.0.14
\ No newline at end of file
+longhornio/support-bundle-kit:v0.0.16
```
Comment (c3y1huang) [inline deploy/longhorn-images.txt:13]: @yangchiu rancher/support-bundle-kit:v0.0.16 published and verified with manual testing.
Please help mirror to longhornio, thanks!

Comment (yangchiu) [reply deploy/longhorn-images.txt:13]: image mirrored: https://hub.docker.com/r/longhornio/support-bundle-kit/tags

**Comment by c3y1huang** [comment]
Test result: https://ci.longhorn.io/job/private/job/longhorn-tests-regression/2615/

**Review by innobead — APPROVED** [review]

**Pull request merged**